### PR TITLE
redact LMS docs

### PR DESCRIPTION
**Pull request by taroface**
Removing MOLT LMS content due to its private preview status.

- Removed the LMS page from the docs repo.
- Removed mentions of LMS (except those already commented-out / hidden) in the docs, including MOLT changelog.
- Removed LMS from MOLT tooling diagram.
- Added redirect from LMS doc to MOLT Overview page.

Note that LMS is still mentioned in the [23.2.0 release notes](http://localhost:4000/docs/releases/v23.2.html#v23-2-0) (when it was released in public preview), but the link will redirect to the MOLT Overview.

### CODE REVIEW

File: src/current/v23.2/migration-overview.md [hunk 1]
```diff
@@ -326,16 +324,14 @@ The following is a high-level overview of the migration steps. The two approache
...
 1. Use [MOLT Fetch]({% link molt/molt-fetch.md %}) to move the source data to CockroachDB. Use the tool to [**replicate ongoing changes**]({% link molt/molt-fetch.md %}#replication) after it performs the initial load of data into CockroachDB. 
 1. As the data is migrating, use [MOLT Verify]({% link molt/molt-verify.md %}) to validate the consistency of the data between the source database and CockroachDB.
-1. After nearly all data from your source database has been moved to CockroachDB (for example, with a <1-second delay or <1000 rows), use MOLT LMS to begin a [*consistent cutover*]({% link molt/live-migration-service.md %}#consistent-cutover) and stop application traffic to your source database. **This begins downtime.**
+1. After nearly all data from your source database has been moved to CockroachDB (for example, with a <1-second delay or <1000 rows), use MOLT LMS to begin a *consistent cutover* and stop application traffic to your source database. **This begins downtime.**
```
Comment (awoods187) [inline src/current/v23.2/migration-overview.md:329]: LMSis still here?

File: src/current/v24.1/migration-overview.md [hunk 2]
```diff
@@ -326,16 +324,14 @@ The following is a high-level overview of the migration steps. The two approache
...
 1. Use [MOLT Fetch]({% link molt/molt-fetch.md %}) to move the source data to CockroachDB. Use the tool to [**replicate ongoing changes**]({% link molt/molt-fetch.md %}#replication) after it performs the initial load of data into CockroachDB. 
 1. As the data is migrating, use [MOLT Verify]({% link molt/molt-verify.md %}) to validate the consistency of the data between the source database and CockroachDB.
-1. After nearly all data from your source database has been moved to CockroachDB (for example, with a <1-second delay or <1000 rows), use MOLT LMS to begin a [*consistent cutover*]({% link molt/live-migration-service.md %}#consistent-cutover) and stop application traffic to your source database. **This begins downtime.**
+1. After nearly all data from your source database has been moved to CockroachDB (for example, with a <1-second delay or <1000 rows), use MOLT LMS to begin a *consistent cutover* and stop application traffic to your source database. **This begins downtime.**
```
Comment (awoods187) [inline src/current/v24.1/migration-overview.md:329]: LMS is here too

**Review by awoods187 — APPROVED** [review]
Thanks for doing this!

File: src/current/v23.2/migration-overview.md (hunk 1 above)
Comment (taroface) [reply src/current/v23.2/migration-overview.md:329]: Fixed, thanks!

File: src/current/v24.1/migration-overview.md (hunk 2 above)
Comment (taroface) [reply src/current/v24.1/migration-overview.md:329]: Fixed.

File: src/current/v23.2/migration-overview.md (hunk 1 above)
Comment (Jeremyyang920) [reply src/current/v23.2/migration-overview.md:329]: I still see MOLT LMS here.

File: src/current/v24.1/migration-overview.md (hunk 2 above)
Comment (Jeremyyang920) [reply src/current/v24.1/migration-overview.md:329]: Same here

File: src/current/v23.2/migration-overview.md (hunk 1 above)
Comment (taroface) [reply src/current/v23.2/migration-overview.md]: I just fixed all 3 of these instances a few moments after you commented -- PTAL!

Comment (Jeremyyang920) [reply src/current/v23.2/migration-overview.md]: LGTM now. Thanks!

**Review by kathancox — APPROVED** [review]
LGTM

**Pull request merged**